examples/skip_rotating_maps.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
What is the difference between this example and the first one?

Reply:
`plotter.rotate(angle=30 * u.deg)` is all I suppose.

Reply:
Can you just add that to the first example?

Reply:
Yeah, I guess that makes more sense